runtime/near-vm-logic/Cargo.toml

```diff
@@ -13,14 +13,21 @@ This crate implements the specification of the interface that Near blockchain ex
 """
 
 [dependencies]
+byteorder = "1.2"
```

Comment:
Why 1.2 if the latest version is 1.3?

Reply:
We use 1.2 everywhere else, so I did not want to bump the version yet in this PR.